mechanics/model/Battle.py

```diff
@@ -19,6 +19,7 @@ class Battle:
     player1: Board
     player2: Board
     initiative_queue: List[PrioritizedItem]
+    distance_to_opponents: Dict
```

Comment:
I've always used either `dict` or `Dict[SomethingHere]`, but I guess that's correct too